getty/center.py

```diff
@@ -278,45 +278,50 @@ def get_expansion_set(go):
 def one_info_pass(
         junit_path, sys_classpath, agent_path, cust_mvn_repo, dyng_go, go, this_hash, target_set,
         changed_methods, changed_tests, inner_dataflow_methods, outer_dataflow_methods, json_filepath):
-    os.sys_call("git checkout " + this_hash)
-    os.sys_call("mvn clean")
+    # os.sys_call("git checkout " + this_hash)
+    # os.sys_call("mvn clean")
 
-    bin_path = mvn.path_from_mvn_call("outputDirectory")
-    test_bin_path = mvn.path_from_mvn_call("testOutputDirectory")
-    cp = mvn.full_classpath(junit_path, sys_classpath, bin_path, test_bin_path)
+    bin_path = mavenCalls.get_bin_path(this_hash)
+    test_bin_path = mavenCalls.get_test_bin_path(this_hash)
+    cp = mavenCalls.get_full_class_path(this_hash, junit_path, sys_classpath, bin_path, test_bin_path)
```

Comment:
can you call it "mvn_adapter" instead of mavenCalls?

Reply:
done!